packages/monitor-v2/src/monitor-og/SnapshotVerification.ts

```diff
@@ -1,19 +1,19 @@
 import { TransactionsProposedEvent } from "@uma/contracts-node/typechain/core/ethers/OptimisticGovernor";
 import assert from "assert";
-import retry from "async-retry";
-import { Options as RetryOptions } from "async-retry";
+import retry, { Options as RetryOptions } from "async-retry";
+import { BigNumber, utils as ethersUtils } from "ethers";
 import fetch from "node-fetch";
 import { request } from "graphql-request";
 import { gql } from "graphql-tag";
 
-import { ethersUtils, MonitoringParams, tryHexToUtf8String } from "./common";
+import { MonitoringParams, tryHexToUtf8String } from "./common";
 
 // If there are multiple transactions within a batch, they are aggregated as multiSend in the mainTransaction.
 interface MainTransaction {
   to: string;
   data: string;
   value: string;
-  operation: string;
+  operation: "0" | "1";
```

Comment:
nit: can you add a quick comment on what 1 and 0 mean here (call and delegatecall?)?